tests/components/switch/test_command_line.py

```diff
@@ -162,16 +162,23 @@ def test_assumed_state_should_be_true_if_command_state_is_false(self):
         """Test with state value."""
         self.hass = get_test_home_assistant()
 
-        # Set state command to false
-        statecmd = False
-
-        no_state_device = command_line.CommandSwitch(self.hass, "Test", "echo",
-                                                     "echo", statecmd, None)
+        # args: hass, device_name, friendly_name, command_on, command_off,
+        #       command_state, value_template
+        init_args = [
+                self.hass,
+                "test_device_name",
+                "Test friendly name!",
+                "echo 'on command'",
+                "echo 'off command'",
+                False,
+                None
+        ]
+
+        no_state_device = command_line.CommandSwitch(*init_args)
```

Comment:
I prefer to write out the expected entity ID. I had it once that the constant was wrong and I didn't notice because I used it in the tests. However the docs made no sense!

For most of our other platform tests we go as a far as to setup the component with the platform passed in as a config. That way you're testing the whole route: config validation, setup and result written to the state machine. However, since the current tests are not like this, I am fine with leaving those as is.